cpp/src/io/orc/aggregate_orc_metadata.cpp

```diff
@@ -151,22 +153,29 @@ aggregate_orc_metadata::aggregate_orc_metadata(
   }
 }
 
-std::vector<metadata::stripe_source_mapping> aggregate_orc_metadata::select_stripes(
+std::tuple<int64_t, size_type, std::vector<metadata::stripe_source_mapping>>
```

Comment:
Not necessarily a change request, but a struct would be more descriptive than a tuple

Reply:
I modified it to be the same as the equivalent code in Parquet. I also prefer struct for return types, but it this case the benefit would be very limited. We only use the return values in a structured binding.

Reply:
> We only use the return values in a structured binding.

Ah, good point.